events/janus_wsevh.c

```diff
@@ -839,3 +828,53 @@ static int janus_wsevh_callback(struct lws *wsi, enum lws_callback_reasons reaso
 	}
 	return 0;
 }
+
+/**
+ * Implements the connecting attempt to the upstream websocket server
+ * sets the connection result (lws_client_connect_info) to static wsi
+ */
+static void janus_wsevh_connect_attempt(void* sul) {
```

Comment:
unused param?

Reply:
i need to have that pointer there for the lws_sul_schedule call. The compiler is otherwise complaining about it.
As i don´t need it and the type is unknown in 3.2 i just use void* (I will add the appropriate comment to the forward decl and the actual impl to point out why)

Reply:
Change it once more to have it better readable.
I use the required lws_sorted_usec_list_t (*) and if not available define it to void (*)